.github/workflows/zip-templates-and-add-to-release.yml

```
@@ -0,0 +1,29 @@
+name: zip-templates-and-add-to-release
+on:
+  release:
+    types: [published]
+jobs:
+  add-templates-to-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Install dependencies
+        run: |
+          npm install -g yarn
+          sudo apt-get update && sudo apt-get install -y --allow-unauthenticated zip
+          yarn install
+      - name: Build templates
+        run: |
+          cd src
+          RELEASE_VERSION=$GITHUB_REF yarn cdn-bundle
+      - name: Zip templates
+        run: zip -r templates.zip templates/*
+      - uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: templates.zip
+          tag: $GITHUB_REF
+          overwrite: true
```

We add a file to a release in the schemas repo using `softprops/action-gh-release` (https://github.com/ONSdigital/eq-questionnaire-schemas/blob/master/.github/workflows/release.yml#L58-L63). We should use the same action here for consistency:
```suggestion
      - name: Release
        uses: softprops/action-gh-release@v1
        with:
          files: templates.zip
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```